src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -118,7 +118,7 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
 
                     if (badTypes.Count != 0)
                     {
-                        _contextAccessor.FhirRequestContext.BundleIssues.Add(
+                        _contextAccessor.FhirRequestContext?.BundleIssues.Add(
```

Comment:
I wonder if SearchOptionsFactory should demand that `_contextAccessor.FhirRequestContext` is not null, thoughts? 
 Should have a standard way to call this from background jobs? @LTA-Thinking

Reply:
I found this issue because the reindex operation runs a search.  In the case where an operation is using the search service, will there be any FhirRequestContext ?  Should we create one for background workers?

Reply:
I noticed we started using it in the export task: https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs#L140

Reply:
Sorry I'm late on this. The request context being null was causing exceptions in Export when we ran into bundle issues. There is a similar problem in Reindex: #1564.

Reply:
This look like it will prevent the exceptions, but the user won't be told that there was an issue.